support-frontend/assets/pages/[countryGroupId]/student/components/StudentHeader.tsx

```diff
@@ -51,6 +52,34 @@ export default function StudentHeader({
 					billingPeriod={BillingPeriod.Annual}
 					cardContent={productContent}
 				/>
+				<GridPicture
+					sources={[
+						{
+							gridId: 'AuStudentLandingHeroMobile',
+							srcSizes: [2000, 1000, 500],
```

Comment:
From this list the size used by the browser depends on the pixel density/zoom level, is that right?